utils/scarb-test-support/src/proc_macro_server.rs

```diff
@@ -193,7 +194,7 @@ impl ProcMacroClient {
         // Usually, we can't discover the ID of the mock package used in test, so we extract it from the PMS response.
         let package_id = response
             .keys()
-            .find(|cu_id| cu_id.starts_with(package_name))
+            .find(|cu_id| cu_id.0.starts_with(package_name))
```

Comment:
```suggestion
            .find(|cu_id| cu_id.name() == package_name)
```

Shouldn't it be this way now?